megamek/src/megamek/common/TankTrailerHitch.java

```diff
@@ -0,0 +1,291 @@
+     * <p>
+     *
+     * @param unit
+     *            - the <code>Entity</code> to be loaded.
+     * @return <code>true</code> if the unit can be loaded, <code>false</code>
+     *         otherwise.
+     */
+    public boolean canLoad(Entity unit) {
+        // Only trailers can be towed.
+        if (!(unit.isTrailer())) {
+            return false;
+        }
+
+        // We must have enough space for the trailer.
+        if (towed != Entity.NONE) {
+            return false;
+        }
+        return true;
+    }
+
+    /**
+     * Load the given unit.
+     *
+     * @param unit
+     *            - the <code>Entity</code> to be loaded.
+     * @exception IllegalArgumentException
+     *                - If the unit can't be loaded, an
+     *                <code>IllegalArgumentException</code> exception will be
+     *                thrown.
+     */
+    public final void load(Entity unit) throws IllegalArgumentException {
+        // If we can't load the unit, throw an exception.
+        if (!canLoad(unit)) {
+            throw new IllegalArgumentException("Can not load " + unit.getShortName() + " onto this hitch.");
+        }
+
+        // Assign the unit as our carried troopers.
+        towed = unit.getId();
+    }
+
+    /**
+     * Get a <code>List</code> of the units currently loaded into this payload.
+     *
+     * @return A <code>List</code> of loaded <code>Entity</code> units. This
+     *         list will never be <code>null</code>, but it may be empty. The
+     *         returned <code>List</code> is independent from the under- lying
+     *         data structure; modifying one does not affect the other.
+     */
+    public final Vector<Entity> getLoadedUnits() {
+        // Return a list of our carried troopers.
+        Vector<Entity> units = new Vector<Entity>(1);
+        if (towed != Entity.NONE) {
+            units.addElement(game.getEntity(towed));
+        }
+        return units;
+    }
+
+    /**
+     * Unload the given unit.
+     *
+     * @param unit
+     *            - the <code>Entity</code> to be unloaded.
+     * @return <code>true</code> if the unit was contain is loadeded in this
+     *         space, <code>false</code> otherwise.
+     */
+    public final boolean unload(Entity unit) {
+        // Are we carrying the unit?
+        Entity trailer = game.getEntity(towed);
+        if ((trailer == null) || !trailer.equals(unit)) {
+            // Nope.
+            return false;
+        }
+
+        // Remove the troopers.
+        towed = Entity.NONE;
+        return true;
+    }
+
+    /**
+     * Return a string that identifies the unused capacity of this transporter.
+     * <p>
+     * Sub-classes should override the <code>getVacancyString</code> method.
+     *
+     * @return A <code>String</code> meant for a human.
+     * @see megamek.common.TankTrailerHitch#getUnusedString()
+     */
+    public final String getUnusedString() {
```

Comment:
consider calling this `getUnusedCapacityString`

Reply:
Same as above. This name is forced on me by Transporter.